modules/nextflow/src/test/groovy/nextflow/scm/BitbucketRepositoryProviderTest.groovy

```diff
@@ -36,7 +36,7 @@ class BitbucketRepositoryProviderTest extends Specification {
         when:
         def url = new BitbucketRepositoryProvider('pditommaso/tutorial',config).getCloneUrl()
```

Comment:
This repo is public, so it doesn't really matter the authentication value used for it. I tested with a private clone in a personal Bitbucket workspace. Fancy making the repo private or using another repo for it, @pditommaso?

Reply:
Makes sense! 👉  6b754459